src/bzfs/bzfs.cxx

```diff
@@ -7666,8 +7667,18 @@ int main(int argc, char **argv)
                             // send client the message that we are ready for him
                             sendUDPupdate(id);
 
+                        unsigned char *buf = ubuf;
+                        uint16_t len;
```

Comment:
this should be declared before use, inside the loop. Declaring it here (and not initializing it) increases complexity.